src/test/resources/com/atomist/rug/test/gherkin/handler/command/SingleMessageFeature1Steps.ts

```diff
@@ -0,0 +1,13 @@
+import {Given,When,Then, HandlerScenarioWorld} from "@atomist/rug/test/handler/Core"
+
+Given("a sleepy country", f => {
+    //console.log("Given invoked for handler")
+})
+When("a visionary leader enters", world => {
+   let handler = world.commandHandler("ReturnsOneMessageCommandHandler")
+   world.invokeHandler(handler, {})
+})
+Then("excitement ensues", world => {
+   console.log("The plan message were " + world.plan().messages())
+    return world.plan().messages().length == 1
```

Comment:
I have a cleanup of the indentation and semicolons in a branch.

https://github.com/atomist/rug/commit/ddba862ba474934ae1aa7d9264ff9705379506ae